sui_core/src/authority_active/gossip/tests.rs

```diff
@@ -43,10 +43,12 @@ pub async fn test_gossip() {
         });
     }
 
+    println!("All authorities started");
     // Let the helper tasks start
-    tokio::task::yield_now().await;
     tokio::time::sleep(Duration::from_secs(1)).await;
+    tokio::task::yield_now().await;
```

Comment:
I noticed if we remove this yield, this test as is does not terminate, but I think the behavior should not change when we put in a yield, is that also what you noticed?